File: 2019-4-29/CanNotExtension/CanNotExtension/ViewController.m
//
//  ViewController.m
//  CanNotExtension
//
//  Created by 吉腾蛟 on 2019/4/30.
//  Copyright © 2019 JiYoung. All rights reserved.
//

#import "ViewController.h"
#import "FileUploadViewController.h"
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    FileUploadViewController *file_VC=[[FileUploadViewController alloc] init];
    [self presentViewController:file_VC animated:YES completion:nil];
}


@end

File: 2019-10-8(Stanford)/PlayingCard/PlayingCard/ViewController.swift
//
//  ViewController.swift
//  PlayingCard
//
//  Created by 吉腾蛟 on 2019/10/11.
//  Copyright © 2019 JY. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    var deck = PlayingCardDeck()
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        
        for _ in 1...10 {
            if let card = deck.draw() {
                print("\(card)")
            }
            
        }
    }


}

